lib/_stream_duplex.js

```diff
@@ -66,6 +66,13 @@ function Duplex(options) {
   }
 }
 
+Duplex.prototype.acquireStandardStream = undefined;
```

Comment:
I think this could be implemented, a Duplex in Whatwg terms is a known entity.

Reply:
please link me to the whatwg duplex interface or equivalent and I'll implement the API for it.

Reply:
It could be just a `TransformStream` with an identity transform, no?

Reply:
Assuming "acquire" means "create", then this could be implemented just by returning `{ readable: new ReadableStream(...), writable: new WritableStream(...) }` where each independently read from the two sides of the Node duplex. I think?